.changeset/serious-geckos-grab.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/radio-input': patch
+---
+
+adjust radio options layout
```

Comment:
Please explain a bit more the changes and how to use the new `additionalContent` prop.

Reply:
Thanks for all the feedback. I will update soon.

Reply:
There are duplicated changesets. Remove one, thanks